internal/edot/pkg/components.go

```diff
@@ -2,7 +2,7 @@
 // or more contributor license agreements. Licensed under the Elastic License 2.0;
 // you may not use this file except in compliance with the Elastic License 2.0.
 
-package otel
+package pkg
```

Comment:
We should probably come up with a better name for this package 😅 . In the otel collector repository it's called `otelcol`, which I think works fine, but I'd be happy with any relatively descriptive name.

Reply:
changed here 81e97fc6ad74265b8c9a2825b9fcddc982c064d7